### ISSUE TITLE
How to PLAY midi with this library

### ISSUE THREAD

**Issue by ragnraok** [jdkoftinoff/jdksmidi#7]
I want to play the midi with this library, but all i can see is parse the midi file, it is possible to play *.mid music with this library?

**Comment by ragnraok** [comment]
BTW, I am using ubuntu 12.04

**Comment by rsp4jack** [comment]
no. you must use any synthesizer library to play midi with this library, like FluidSynth, EAS, etc..
